### ISSUE TITLE
Display descriptions of schemas, classes and elements in a request

### ISSUE THREAD

**Issue by pjmonks** [MauroDataMapper/mdm-explorer#272]
# Description

Add the descriptions (if available) for each schema, class and element listed in a data request to help the user understand the contents of what they are requesting.

# Additional context

This is a rough mockup of what could be shown, but would need better styling:

![image](https://user-images.githubusercontent.com/3219480/219096436-a8d2145d-3a78-44b6-abb1-908febb09da9.png)

**Comment by NigelPalmer** [comment]
@pjmonks, I don't think this solves the problem that Jim and James raised. My understanding is that they were using the Query Builder which presents them with a simple list of field names. At that point they are unable to see any additional information, and it is on that page that they would wish to see the information. Putting the descriptions on the lists of fields does not aid them in writing the query because it is on a different page. We need to work out whether we can get this information to the query builder page.

**Comment by NigelPalmer** [comment]
I am movin
